ovs/extensions/plugins/albacli.py

```diff
@@ -81,26 +105,27 @@ def run(command, **kwargs):
                     if exit_code != 0:  # Raise same error as check_output
                         raise CalledProcessError(exit_code, cmd_string, output)
                 else:
-                    if debug:
+                    if debug is True:
                         output, stderr = client.run(cmd_list, debug=True)
                         debug_log.append('stderr: {0}'.format(stderr))
                     else:
                         output = client.run(cmd_list, debug=False).strip()
                     debug_log.append('stdout: {0}'.format(output))
-            except CalledProcessError as ex:
-                if to_json is True:
-                    output = json.loads(ex.output)
-                    raise RuntimeError(output.get('error', {}).get('message'))
-                raise
-            duration = time.time() - start
-            if duration > 0.5:
-                logger.warning('AlbaCLI call {0} took {1}s'.format(command, round(duration, 2)))
-            if to_json is True:
+
                 output = json.loads(output)
-                if output['success'] is True:
-                    return output['result']
-                raise RuntimeError(output['error']['message'])
-            return output
+                duration = time.time() - start
+                if duration > 0.5:
+                    logger.warning('AlbaCLI call {0} took {1}s'.format(command, round(duration, 2)))
+            except CalledProcessError as cpe:
+                try:
+                    output = json.loads(cpe.output)
+                except Exception:
+                    raise Exception('Failed to JSON decode. Output: {0}\nCommand: {1}'.format(cpe.output, cmd_string))
```

Comment:
I'd rather not mention the JSON stuff as strictly speaking we don't expect JSON here. Just raise a ```RuntimeError``` with the ```cpe.output```. If we can however parse as JSON, it's just a nice-to-have.